apps/browser-extension-wallet/src/views/browser-view/features/assets/components/AssetsPortfolio/AssetsPortfolio.tsx

```diff
@@ -118,25 +152,38 @@ export const AssetsPortfolio = ({
           }}
         />
       )}
-      <Skeleton loading={isPortfolioBalanceLoading || !assetList}>
-        {portfolioBalanceAsBigNumber.gt(0) ? (
-          <AssetTable
-            rows={assetList}
-            onRowClick={onRowClick}
-            totalItems={totalAssets}
-            scrollableTargetId={isPopupView ? CONTENT_LAYOUT_ID : LACE_APP_ID}
-            onLoad={onTableScroll}
-            popupView={isPopupView}
-          />
-        ) : (
-          <FundWalletBanner
-            title={t('browserView.assets.welcome')}
-            subtitle={t('browserView.assets.startYourWeb3Journey')}
-            prompt={t('browserView.fundWalletBanner.prompt')}
-            walletAddress={walletInfo.addresses[0].address.toString()}
-          />
-        )}
-      </Skeleton>
+      {assetList?.length > SEARCH_ASSET_LENGTH && (
+        <SearchBox
+          placeholder={t('browserView.assets.searchPlaceholder')}
+          onChange={(text) => handleSearch(text)}
+          data-testid="assets-search-input"
+          value={searchValue}
+          onClear={() => setSearchValue('')}
+        />
+      )}
+      {currentAssets.total > 0 ? (
```

Comment:
Suggestion: turn each ternary into two separate conditions. IMO now when you scroll down to the "else" markup its harder to locate the condition for which it gets rendered, and here it is even harder because there are two ternaries nested, it would be nicer if the condition was standing just before the markup.

```suggestion
      {currentAssets.total > 0 && (
```

Reply:
Fixed on 55a094f